velox/exec/OutputBuffer.h

```diff
@@ -254,7 +266,28 @@ class OutputBuffer {
     /// Stats of the OutputBuffer's destinations.
     std::vector<DestinationBuffer::Stats> buffersStats;
 
-    std::string toString() const;
+    std::string toString() {
+      std::string destinationBufferStats;
+      if (!buffersStats.empty()) {
+        for (auto& destinationBufferStat : buffersStats) {
+          destinationBufferStats += destinationBufferStat.toString();
```

Comment:
Do we want one line per each destination buffer given there would be hundreds of them for better display? Thanks!

Reply:
Yes I have updated this PR to one line per destination buffer